linkatos.py

```diff
@@ -2,18 +2,35 @@
 import os
 import time
 from slackclient import SlackClient
+import pyrebase
 import linkatos.parser as parser
 import linkatos.confirmation as confirmation
 import linkatos.printer as printer
 import linkatos.utils as utils
+import linkatos.firebase as fb
 
-# starterbot's ID as an environment variable
+# starterbot environment variables
 BOT_ID = os.environ.get("BOT_ID")
 SLACK_BOT_TOKEN = os.environ.get("SLACK_BOT_TOKEN")
 
 # instantiate Slack clients
 slack_client = SlackClient(SLACK_BOT_TOKEN)
 
+# firebase environment variables
+FB_API_KEY = os.environ.get("FB_API_KEY")
+FB_USER = os.environ.get("FB_USER")
+FB_PASS = os.environ.get("FB_PASS")
+
+# initialise firebase
```

Comment:
Why don't you wrap this into a function and relocate it inside the `firebase` module? Having things in the outer scope is not as transportable nor testable.

```python
def initialise(api_key):
  config = {
    "apiKey": api_key,
    "authDomain": "coses-acbe6.firebaseapp.com",
    "databaseURL": "https://coses-acbe6.firebaseio.com",
    "storageBucket": "coses-acbe6.appspot.com"}

  return pyrebase.initialize_app(config)
```

Note: I used british spelling because our conversation on how to spell licence.

This function easily allows you to, later on do the following:


```python
def initialise(api_key, project_name):
  config = {
    "apiKey": api_key,
    "authDomain": "{}.firebaseapp.com".format(project_name),
    "databaseURL": "https://{}.firebaseio.com".format(project_name),
    "storageBucket": "{}.appspot.com".format(project_name),
  }

  return pyrebase.initialize_app(config)
```